### DISCUSSION TITLE
Transonic 2-D Stage (turbomachinery case)

### DISCUSSION THREAD

**Discussion by safdarianebi** [su2code/SU2#2301] [Q&A]
I am trying to simulate a 2-D transonic turbomachinery stage. And, the inlet mach number is equal to 1.2. I am using MARKER_GILES and MARKER_SUPERSONIC_INLET boundary conditions. My simulation converges perfectly, but the final result is not satisfying as inflow mach number is 0.4.
I don't know what is wrong, and I would be grateful if you could help me solve this problem.

**Comment by joshkellyjak** [comment]
Are you defining two boundary conditions on the same boundary?
